_data/navigation.yml

```diff
@@ -204,96 +206,24 @@ t:
         url: /cn/docs/quick-start-dataconsistency/
 
     users:
-      - title: 用户手册
+      - title: Java Chassis
         children:
           - title: Java Chassis系统架构
```

Comment:
I guess you just want to keep a section to briefly introduce the sub projects of servicecomb, but not a “java-chassis architecture”, right?

Reply:
Right. My original idea was to keep the same style as before.